### DISCUSSION TITLE
Missing image hash on Docs

### DISCUSSION THREAD

**Discussion by Mecanik** [hbstack/.github#7] [Issues and bugs]
Hi,

Bugs:

- The Docs is missing image hashing (when you click to view the full image)
- The generated images across the projects are not consistent in filenames; the gallery for example, thumbnails generate files like: FILE_hu84ccfb793eb153d5f929caaf8b63424a_6189003_f3f72e4e599b207dacd59f173c67ac1a.webp and once you click to view the image FILE_hu84ccfb793eb153d5f929caaf8b63424a_6189003_3840x2160_resize_q95_h2_box.webp.

Suggestion:

- Each generate images, in every area, should have consistent file names and format
- Thumbnails and featured (small) images can be resided via Hugo, but large/full view images should be original high quality and only contain a hash for caching.

Thanks

**Comment by razonyang** [comment]
Should be fixed by https://github.com/hugomods/images/pull/6
